README.Rmd

```diff
@@ -0,0 +1,153 @@
+)
+
+# just to keep the code clean below
+suppressWarnings(suppressPackageStartupMessages(library(dplyr)))
+```
+
+# Scales <img src="man/figures/logo.png" align="right" />
+
+[![Build Status](https://travis-ci.org/r-lib/scales.svg?branch=master)](https://travis-ci.org/r-lib/scales)
+[![Coverage Status](https://img.shields.io/codecov/c/github/r-lib/scales/master.svg)](https://codecov.io/github/r-lib/scales?branch=master)
+[![CRAN_Status_Badge](http://www.r-pkg.org/badges/version/scales)](https://cran.r-project.org/package=scales)
+
+# Overview
+
+One of the most difficult parts of any graphics package is scaling, 
+converting from data values to perceptual properties. The inverse of scaling,
+making guides (legends and axes) that can be used to read the graph, is often 
+even harder! The scales packages provides the internal scaling infrastructure to
+[ggplot2](github.com/tidyverse/ggplot2) and its functions allow users to 
+customize the transformations, breaks, guides and palettes used in visualizations.
+
+The idea of the scales package is to implement scales in a way 
+that is graphics system agnostic, so that everyone can benefit by pooling 
+knowledge and resources about this tricky topic.
+
+# Installation
+
+```{r, eval = FALSE}
+# Scales is installed when you install ggplot2 or the tidyverse.
+# But you can install just scales from CRAN:
+install.packages("scales")
+
+# Or the development version from Github:
+# install.packages("devtools")
+devtools::install_github("r-lib/scales")
+```
+  
+# Usage
+```{r}
+library(scales)
+```
+
+### Formatters 
+
+Outside of ggplot2 where it powers all the aesthetic scales, axes formatting, 
+and data transformations internally, the scales package also provides useful helper 
+functions for formatting numeric data for all types of presentation:
+
+```{r}
+# percent() function takes a numeric and does your division and labelling for you
+percent(seq(.1, .7, .1))
+
+# comma() can add commas into large numbers for easier readability
+comma(10e6)
+
+# dollar() can help you present currrency
+dollar(c(100, 125, 3000))
+
+# unit_format() can help you handle unique units
+# the scale argument doing the conversion on the fly
+unit_format(unit = "ha", scale = 1e-4)(c(10e6, 10e4, 8e3))
+```
+
+All these formatters allow further customization, especially useful for meeting
+diverse international standards:
+
+```{r}
+# for instance European number formatting is easily set:
+number(c(12.3, 4, 12345.789, 0.0002),
+  accuracy = .0001, big.mark = ".", decimal.mark = ","
+)
+
+# or percent formatting in the French style
+french_percent <- percent_format(
+  decimal.mark = ",",
+  suffix = " %"
+)
+french_percent(runif(10))
+
+# or currency formatting Euros (and simple conversion!)
+usd_to_euro <- dollar_format(prefix = "", suffix = "\u20ac", scale = .85)
+usd_to_euro(100)
+```
+
+### Colour palettes
+
+scales also provides functions to define colour and other aesthetic scales, e.g.:
```

Comment:
These are used to power the scales in ggplot2, but you can use in any plotting system. The following example shows how you might use with a base plot.